native_words.asm

```diff
@@ -899,11 +899,11 @@ xt_at_xy:
 
                 lda #AscESC
                 jsr emit_a
-                lda #$5B        ; ASCII for "["
+                lda #'['
```

Comment:
This is also great.  I vaguely remember having an argument with character literals with the previous assembler we were using, but 64tass has no issue and this format works with most other assemblers I have experience with.